### PR TITLE
34-create-office-map-and-workspaces-draft-controller-witt-empty-methods

### PR DESCRIPTION
**Pull request by JmlKhan**

### CODE REVIEW

**Comment by JmlKhan** [comment]
@IrakliMatcharadze

File: TelegramBotAPI/Controllers/WorkplaceController.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+﻿using DataLayer.Models;
+//using Microsoft.AspNetCore.Http;
```
Comment (apilatau) [inline TelegramBotAPI/Controllers/WorkplaceController.cs]: remove commented code

**Review by apilatau — APPROVED** [review]
after resolving comment it is possible to approve

File: TelegramBotAPI/Controllers/WorkplaceController.cs (hunk 1 above)
Comment (JmlKhan) [reply TelegramBotAPI/Controllers/WorkplaceController.cs]: Resolved

**Review by apilatau — APPROVED** [review]

**Pull request merged**